rh-summit-2018/module-01.adoc

```diff
@@ -270,12 +267,15 @@ Click on the "Add data source" button from the Grafana home in order to add Prom
 image::grafana_home.png[grafana home]
 
 Fill in the information about the Prometheus data source, specifying a name and "Prometheus" as type.
-In the URL field, the connection string to the Prometheus server `http://prometheus:9090`` should be specified.
+In the URL field, use `http://prometheus.l1099-kafka.svc.cluster.local:9090` as the URL to the Prometheus server.
```

Comment:
Isn't it enough specifying just the service name ("prometheus") and not the complete name with namespace ? Did you try and it didn't work ?

Reply:
Yeah, both are correct. Either way works for me.